sklearn/base.py

```diff
@@ -83,7 +88,6 @@ def clone(estimator, *, safe=True):
         new_object_params[name] = clone(param, safe=False)
     new_object = klass(**new_object_params)
     params_set = new_object.get_params(deep=False)
-
     # quick sanity check of the parameters of the clone
```

Comment:
Removing this blank line is unrelated to the main purpose of this PR (the docstring improvement). Please revert this change by accepting the suggestion below (and pulling the commit locally).

```suggestion

    # quick sanity check of the parameters of the clone
```

Reply:
I removed it.

Reply:
Thanks for the feedback @ogrisel - I updated the pull request. Could you please check my work?